python/pearson_matrix.py

```diff
@@ -0,0 +1,96 @@
+"""
+Write a function to load your data. I'll be using a random dataset with approximately the same size as the CCLE H3 relative values proteomics dataset. The histone marks and genes are the rows, while the columns are the cell line values. You can perform numpy calculations on pandas dataframes, but you will have to figure out how to do that on your own.
```

Comment:
If you don't need this script anymore, you can delete it. If you want to keep it, you can edit your .gitignore file in the top of the egem folder and type in the path to the file. That will cause git to not add this file when pushing.

Reply:
Going to delete soon just at the end to be sure